_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -65816,7 +65816,9 @@
 /turf/simulated/floor/plating,
 /area/station/maintenance/port)
 "jRo" = (
-/obj/effect/spawner/airlock/s_to_n/long/square/engineer,
+/obj/effect/spawner/airlock/s_to_n/long/square/engineer{
+	req_access_txt = null
```

Comment:
I tested it, you want to make it so anyone, even those without ID, can use these airlocks?

Reply:
yes anyone can use them